docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.6'
+
+services:
+  etl-nwis:
```

Comment:
Should line 4 be `etl-biodata`?

Reply:
Seems highly probable that, indeed, it should be :)  Fixed